content/blog/v1-40-released.md

```diff
@@ -0,0 +1,55 @@
+date: null
+author: DNin01
+---
+
+Before we cover this update, let's take a quick detour and review the last hotfix.
+
+# In case you missed it: v1.39.2 and addon removals
+
+The v1.39.2 hotfix was released recently and resolves a few glitches that popped up after some Scratch updates. If you had issues with [Sprite folders](https://scratch.mit.edu/scratch-addons-extension/settings#addon-folders) or [custom color themes](https://scratch.mit.edu/scratch-addons-extension/settings#addon-editor-dark-mode), those have been fixed in this version.
+
+Since Scratch now has its own sprite deletion confirmation dialog and the discussion forums now use 3.0 scratchblocks, the addons for those have been removed, because now everyone can enjoy those features! 👏
+
+# What's new in v1.40
+
+We've made a few improvements to existing addons and there's a new addon or two!
+
+## 🌻 Project splash screens
+
+On project pages, the new addon [Show project thumbnail while loading](https://scratch.mit.edu/scratch-addons-extension/settings#addon-player-thumb) adds a splash screen with the thumbnail of the project while it's loading, replacing the solid blue loading screen.
+
+![The loading screen for a Scratch project with a faint image of the project's thumbnail as the background](/assets/img/blog/v1-40-released/project-splash.png)
+
+## 🗑️ Keep the trash can open
+
+Another new addon, [Delete sprites without confirmation](https://scratch.mit.edu/scratch-addons-extension/settings#addon-no-sprite-confirm), **disables** Scratch's confirmation for deleting sprites. You know, if you often find yourself deleting bunches of sprites and don't want to be asked every time. Just make sure you don't delete anything you didn't mean to!
+
+## 📸 Block snapshot options
+
+When taking a snapshot of blocks on the code area with [Save blocks as image](https://scratch.mit.edu/scratch-addons-extension/settings#addon-blocks2image), you'll now be presented with a dialog, showing you a preview of the snapshot which you can choose whether you want to copy to the clipboard or save to a file.
+
+![A dialog menu showing a snapshot of Scratch blocks with buttons to save and copy the snapshot](/assets/img/blog/v1-40-released/block-image-dialog.png)
+
+## 🔍 The search bar moved
+
+The [Sprite pane search](https://scratch.mit.edu/scratch-addons-extension/settings#addon-search-sprites) bar has changed shape and moved next to the add sprite button.
+
+<video src="/assets/img/blog/v1-40-released/search-bar.mp4" controls type="video/mp4" autoplay loop></video>
+
+## ✨ Other improvements, changes, and bug fixes
+
+- The tab bar in the extension's popup has been redesigned and "Addons" is now the first tab.
+- The "Do not disturb" dropdown has been updated with options to enable for 4 hours and until the next day.
+- [Customizable editor menu bar](https://scratch.mit.edu/scratch-addons-extension/settings#addon-custom-menu-bar) is now able to hide Scratch's new Debug button.
```

Comment:
I'll mention MP4 support if the plan is to [fix the audio recording bug](https://github.com/ScratchAddons/ScratchAddons/pull/8024) for v1.40.1.
```suggestion
- [Customizable editor menu bar](https://scratch.mit.edu/scratch-addons-extension/settings#addon-custom-menu-bar) is now able to hide Scratch's new Debug button.
- [Project video recorder](https://scratch.mit.edu/scratch-addons-extension/settings#addon-mediarecorder) now supports saving MP4 recordings on Chrome.
```

Reply:
> fix the audio recording bug for v1.40.1

I'm not sure yet.

Reply:
> > fix the audio recording bug for v1.40.1
> 
> I'm not sure yet.

_\*sigh\*_

At least this bug has been around since v1.39.2 so I guess there's no rush.

Reply:
Exactly

There's also a reasonable chance v1.40.0 has some medium severity bug no one has reported yet due to Christmas

Reply:
ScratchAddons/ScratchAddons#8024 has been merged thus MP4 export support lives on, so it should be okay to announce that we added MP4 support.